tests/gtest/avifrgbtoyuvthreadingtest.cc

```diff
@@ -0,0 +1,124 @@
+// Copyright 2022 Google LLC
```

Comment:
2024

Reply:
The file was copied in the first commit so we should keep the copyright as is I think.